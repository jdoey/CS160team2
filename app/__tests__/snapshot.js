import {render} from '@testing-library/react'
import Page from '../page'

it('No changes in Landing Page', () => {
    const { container } = render(<Page />)
  expect(container).toMatchSnapshot()
})