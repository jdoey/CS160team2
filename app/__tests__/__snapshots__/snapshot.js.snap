// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`No changes in Landing Page 1`] = `
<div>
  <div
    class="css-0"
  >
    <div
      class="css-8lthux"
    >
      <div
        class="css-1vakep4"
      >
        <button
          aria-label="Toggle Navigation"
          class="chakra-button css-p7mkev"
          type="button"
        >
          <svg
            aria-hidden="true"
            class="chakra-icon css-1etxpkf"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M 3 5 A 1.0001 1.0001 0 1 0 3 7 L 21 7 A 1.0001 1.0001 0 1 0 21 5 L 3 5 z M 3 11 A 1.0001 1.0001 0 1 0 3 13 L 21 13 A 1.0001 1.0001 0 1 0 21 11 L 3 11 z M 3 17 A 1.0001 1.0001 0 1 0 3 19 L 21 19 A 1.0001 1.0001 0 1 0 21 17 L 3 17 z"
              fill="currentColor"
            />
          </svg>
        </button>
      </div>
      <div
        class="css-8k9ono"
      >
        <a
          class="chakra-button css-1izydtr"
          href="/"
        >
          <img
            alt="Maze Bank Logo"
            data-nimg="1"
            decoding="async"
            height="40"
            loading="lazy"
            src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
            srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
            style="color: transparent;"
            width="40"
          />
          MAZE BANK
        </a>
        <div
          class="css-nmdqie"
        >
          <div
            class="chakra-stack css-fhxiwe"
          >
            <div
              class="css-0"
            >
              <a
                aria-controls="popover-content-:r2:"
                aria-expanded="false"
                aria-haspopup="dialog"
                class="css-1t6f37y"
                href="/ATM"
                id="popover-trigger-:r2:"
              >
                ATM Locations
              </a>
            </div>
          </div>
        </div>
      </div>
      <div
        class="chakra-stack css-9kmm5a"
      >
        <a
          class="chakra-button css-oulk1f"
          href="/signin"
        >
          Sign In
        </a>
        <a
          class="chakra-button css-4qulu8"
          href="/enroll"
        >
          Enroll
        </a>
      </div>
    </div>
    <div
      class="chakra-collapse"
      style="overflow: hidden; display: none; opacity: 0; height: 0px;"
    >
      <div
        class="chakra-stack css-1gbq759"
      >
        <div
          class="chakra-stack css-1kkt86i"
        >
          <a
            class="css-11dfkyq"
            href="/ATM"
          >
            <p
              class="chakra-text css-y33gq0"
            >
              ATM Locations
            </p>
          </a>
          <div
            class="chakra-collapse"
            style="overflow: hidden; display: none; opacity: 0; height: 0px;"
          >
            <div
              class="chakra-stack css-hcix0d"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="css-1f4bi0b"
  >
    <div
      class="chakra-stack css-y2jo52"
    >
      <h2
        class="chakra-heading css-1awp0jh"
      >
        We'll keep your money safe 
        <br />
        <span
          class="chakra-text css-sz6yka"
        >
          ... probably.
        </span>
      </h2>
      <div
        class="chakra-stack css-1pe0sil"
      >
        <a
          class="chakra-button css-1s66jqo"
          href="/enroll"
        >
          Enroll Now
        </a>
      </div>
    </div>
  </div>
  <div
    class="css-1qieuwx"
  >
    <div
      class="css-sb8y84"
    >
      <h2
        class="chakra-heading css-3oqufc"
      >
        Choose what's right for you
      </h2>
      <div
        class="css-gwe340"
      >
        <div
          class="chakra-stack css-tl3ftk"
        >
          <div
            class="css-ex2k6o"
          >
            <svg
              class="chakra-icon css-zzjix2"
              fill="currentColor"
              focusable="false"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 640 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M0 448c0 17.67 14.33 32 32 32h576c17.67 0 32-14.33 32-32V128H0v320zm448-208c0-8.84 7.16-16 16-16h96c8.84 0 16 7.16 16 16v32c0 8.84-7.16 16-16 16h-96c-8.84 0-16-7.16-16-16v-32zm0 120c0-4.42 3.58-8 8-8h112c4.42 0 8 3.58 8 8v16c0 4.42-3.58 8-8 8H456c-4.42 0-8-3.58-8-8v-16zM64 264c0-4.42 3.58-8 8-8h304c4.42 0 8 3.58 8 8v16c0 4.42-3.58 8-8 8H72c-4.42 0-8-3.58-8-8v-16zm0 96c0-4.42 3.58-8 8-8h176c4.42 0 8 3.58 8 8v16c0 4.42-3.58 8-8 8H72c-4.42 0-8-3.58-8-8v-16zM624 32H16C7.16 32 0 39.16 0 48v48h640V48c0-8.84-7.16-16-16-16z"
              />
            </svg>
          </div>
          <p
            class="chakra-text css-1azu20y"
          >
            Checking
          </p>
        </div>
        <div
          class="chakra-stack css-tl3ftk"
        >
          <div
            class="css-ex2k6o"
          >
            <svg
              class="chakra-icon css-zzjix2"
              fill="currentColor"
              focusable="false"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 576 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M560 224h-29.5c-8.8-20-21.6-37.7-37.4-52.5L512 96h-32c-29.4 0-55.4 13.5-73 34.3-7.6-1.1-15.1-2.3-23-2.3H256c-77.4 0-141.9 55-156.8 128H56c-14.8 0-26.5-13.5-23.5-28.8C34.7 215.8 45.4 208 57 208h1c3.3 0 6-2.7 6-6v-20c0-3.3-2.7-6-6-6-28.5 0-53.9 20.4-57.5 48.6C-3.9 258.8 22.7 288 56 288h40c0 52.2 25.4 98.1 64 127.3V496c0 8.8 7.2 16 16 16h64c8.8 0 16-7.2 16-16v-48h128v48c0 8.8 7.2 16 16 16h64c8.8 0 16-7.2 16-16v-80.7c11.8-8.9 22.3-19.4 31.3-31.3H560c8.8 0 16-7.2 16-16V240c0-8.8-7.2-16-16-16zm-128 64c-8.8 0-16-7.2-16-16s7.2-16 16-16 16 7.2 16 16-7.2 16-16 16zM256 96h128c5.4 0 10.7.4 15.9.8 0-.3.1-.5.1-.8 0-53-43-96-96-96s-96 43-96 96c0 2.1.5 4.1.6 6.2 15.2-3.9 31-6.2 47.4-6.2z"
              />
            </svg>
          </div>
          <p
            class="chakra-text css-1azu20y"
          >
            Savings
          </p>
        </div>
      </div>
    </div>
  </div>
  <div
    class="css-yaccxs"
  >
    <div
      class="chakra-stack chakra-container css-1lb86fd"
    >
      <div
        class="chakra-stack css-dk69dq"
      >
        <h2
          class="chakra-heading css-0"
        >
          Hear from our Customers
        </h2>
      </div>
      <div
        class="chakra-stack css-13rngpi"
      >
        <div
          class="css-0"
        >
          <div
            class="chakra-stack css-m72sps"
          >
            <h3
              class="chakra-heading css-6dvxm2"
            >
              Fees
            </h3>
            <p
              class="chakra-text css-1cuin5z"
            >
              I can't believe how much I've been nickel and dimed by this bank. Every time I turn around, there's another fee waiting for me. It's like they're playing a game to see how much money they can take from me without me noticing. Shameful!
            </p>
          </div>
          <div
            class="css-w6zqsg"
          >
            <span
              class="chakra-avatar css-ggnqxp"
            >
              <svg
                aria-label=" avatar"
                class="chakra-avatar__svg css-16ite8i"
                role="img"
                viewBox="0 0 128 128"
              >
                <path
                  d="M103,102.1388 C93.094,111.92 79.3504,118 64.1638,118 C48.8056,118 34.9294,111.768 25,101.7892 L25,95.2 C25,86.8096 31.981,80 40.6,80 L87.4,80 C96.019,80 103,86.8096 103,95.2 L103,102.1388 Z"
                  fill="currentColor"
                />
                <path
                  d="M63.9961647,24 C51.2938136,24 41,34.2938136 41,46.9961647 C41,59.7061864 51.2938136,70 63.9961647,70 C76.6985159,70 87,59.7061864 87,46.9961647 C87,34.2938136 76.6985159,24 63.9961647,24"
                  fill="currentColor"
                />
              </svg>
            </span>
            <div
              class="chakra-stack css-ylfx56"
            >
              <p
                class="chakra-text css-35ezg3"
              >
                Michael De Santa
              </p>
              <p
                class="chakra-text css-130t7v7"
              >
                Former Bank Robber
              </p>
            </div>
          </div>
        </div>
        <div
          class="css-0"
        >
          <div
            class="chakra-stack css-m72sps"
          >
            <h3
              class="chakra-heading css-6dvxm2"
            >
              Intuitive Design
            </h3>
            <p
              class="chakra-text css-1cuin5z"
            >
              I can't say enough about the seamless experience I've had using the banking app from Maze Bank. It's truly a game-changer in teh world of mobile banking. From the moment I began using it, I was impressed by its intuitive design and user-friendly interface.
            </p>
          </div>
          <div
            class="css-w6zqsg"
          >
            <span
              class="chakra-avatar css-ggnqxp"
            >
              <svg
                aria-label=" avatar"
                class="chakra-avatar__svg css-16ite8i"
                role="img"
                viewBox="0 0 128 128"
              >
                <path
                  d="M103,102.1388 C93.094,111.92 79.3504,118 64.1638,118 C48.8056,118 34.9294,111.768 25,101.7892 L25,95.2 C25,86.8096 31.981,80 40.6,80 L87.4,80 C96.019,80 103,86.8096 103,95.2 L103,102.1388 Z"
                  fill="currentColor"
                />
                <path
                  d="M63.9961647,24 C51.2938136,24 41,34.2938136 41,46.9961647 C41,59.7061864 51.2938136,70 63.9961647,70 C76.6985159,70 87,59.7061864 87,46.9961647 C87,34.2938136 76.6985159,24 63.9961647,24"
                  fill="currentColor"
                />
              </svg>
            </span>
            <div
              class="chakra-stack css-ylfx56"
            >
              <p
                class="chakra-text css-35ezg3"
              >
                Franklin Clinton
              </p>
              <p
                class="chakra-text css-130t7v7"
              >
                Repossession Agent
              </p>
            </div>
          </div>
        </div>
        <div
          class="css-0"
        >
          <div
            class="chakra-stack css-m72sps"
          >
            <h3
              class="chakra-heading css-6dvxm2"
            >
              Customer Service
            </h3>
            <p
              class="chakra-text css-1cuin5z"
            >
              "If you enjoy spending hours on hold waiting to speak to a representative who can't actually help you, then by all means, bank with this institution. Otherwise, do yourself a favor and find a bank that values your time and treats you like a human being."
            </p>
          </div>
          <div
            class="css-w6zqsg"
          >
            <span
              class="chakra-avatar css-ggnqxp"
            >
              <svg
                aria-label=" avatar"
                class="chakra-avatar__svg css-16ite8i"
                role="img"
                viewBox="0 0 128 128"
              >
                <path
                  d="M103,102.1388 C93.094,111.92 79.3504,118 64.1638,118 C48.8056,118 34.9294,111.768 25,101.7892 L25,95.2 C25,86.8096 31.981,80 40.6,80 L87.4,80 C96.019,80 103,86.8096 103,95.2 L103,102.1388 Z"
                  fill="currentColor"
                />
                <path
                  d="M63.9961647,24 C51.2938136,24 41,34.2938136 41,46.9961647 C41,59.7061864 51.2938136,70 63.9961647,70 C76.6985159,70 87,59.7061864 87,46.9961647 C87,34.2938136 76.6985159,24 63.9961647,24"
                  fill="currentColor"
                />
              </svg>
            </span>
            <div
              class="chakra-stack css-ylfx56"
            >
              <p
                class="chakra-text css-35ezg3"
              >
                Trevor Phillips
              </p>
              <p
                class="chakra-text css-130t7v7"
              >
                CEO of Trevor Phillips Industries
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
